src/components/question/Question.styles.ts

```diff
@@ -0,0 +1,95 @@
+import palette from '~/styles/app-theme/app.pallete'
+import { TypographyVariantEnum } from '~/types'
+
+const actionIconWrapper = {
+  display: 'flex',
+  alignItems: 'center'
+}
+
+const actionIcon = {
+  fontSize: '18px',
+  mr: '10px'
+}
+
+export const styles = {
+  root: {
+    width: '60%',
+    background: palette.basic.white,
```

Comment:
do you need to import palette here? 
Maybe you can use backgroundColor: 'basic.white'